_config.yml

```diff
@@ -43,13 +45,8 @@ url: https://analytics.usa.gov
 # Default dropdown title, will not appear if there are no sub-pages
 dropdown_title: All Participating Websites
 
-# Default data url
-#data_url: https://s3.amazonaws.com/cg-ff27f612-cd16-4a76-94f8-628986af2158/data/
-data_url: https://s3-us-gov-west-1.amazonaws.com/cg-ff27f612-cd16-4a76-94f8-628986af2158/data/live
-
-# GitHub information
-org_name: GSA
-repo_name: analytics.usa.gov
+# Default data url (also needs to be in nginx.conf for cloud.gov deployment)
+data_url: https://analytics.usa.gov/data
```

Comment:
The `data_url` is now a base URL for all data URLs, of the front page and the agency pages. Each page is responsible for prepending its own prefix, be it `/live` or `/:slug`.